### DISCUSSION TITLE
Top Language Card | exclude_repo param isn't accounted for under eight-theta path

### DISCUSSION THREAD

**Discussion by roeishc** [anuraghazra/github-readme-stats#3829] [Q&A] [answered]
### Describe the bug

Under the following path:
https://github-readme-stats-eight-theta.vercel.app/api/top-langs/?username=roeishc&exclude_repo=human-activity-recognition
The repository that I added to exclude_repo is still accounted for.

However, under the following path:
https://github-readme-stats.vercel.app/api/top-langs/?username=roeishc&exclude_repo=human-activity-recognition
The repository is ignored.

### Expected behavior

Repository language(s) ignored in both paths.

### Screenshots / Live demo link

Not working (with eight-theta in path):
<img src="https://github-readme-stats-eight-theta.vercel.app/api/top-langs/?username=roeishc&exclude_repo=human-activity-recognition" />
screenshot in case the markdown image doesn't work:
![image](https://github.com/anuraghazra/github-readme-stats/assets/95538414/9f912e07-4178-4d32-be2d-18e4f846f496)
<br><br>
Working (without eight-theta in path):
<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=roeishc&exclude_repo=human-activity-recognition" />
screenshot in case the markdown image doesn't work:
![image](https://github.com/anuraghazra/github-readme-stats/assets/95538414/7aafca9f-0911-4cf0-9c92-dd48ade1e161)

### Additional context

I'm new to this repository (which is amazing by the way, thanks!), so maybe the eight-theta path is deprecated and I'm not aware.

**Comment by qwerty541** [comment, marked as answer]
Hey @roeishc,

https://github-readme-stats.vercel.app/ - it is our official public deployment which is always up to date.

https://github-readme-stats-eight-theta.vercel.app/ - Looks like it is some user's own outdated deployment, it is not related to our team in any way.

I recommend you to avoid using some thirdparty deployments in favour of public or your own one.
